modules/serverless-functions-all-values-in-configmap-to-env-variables.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-all-values-in-configmap-to-env-variables_{context}"]
+= Setting environment variables from all values defined in a config map
+
```

Comment:
Should you add an intro paragraph to each of the files in this PR, perhaps with a reason/benefit for doing the action? _You can do {blah}, which allows you to {yadda, yadda}..._

Reply:
For all of these small sections, this introduction:

https://deploy-preview-34089--osdocs.netlify.app/openshift-enterprise/latest/serverless/functions/serverless-functions-accessing-secrets-configmaps#serverless-functions-secrets-configmaps-manually_serverless-functions-secrets

is enough, and the specifics of each particular section is reflected in the section's name